activejdbc/src/test/resources/oracle_schema.sql

```diff
@@ -1087,7 +1087,7 @@ end;
 
 
 
-CREATE TABLE passengers (id NUMBER NOT NULL, vehicle VARCHAR(56), mode VARCHAR(56), user_id NUMBER)
+CREATE TABLE passengers (id NUMBER NOT NULL, vehicle VARCHAR(56), "mode" VARCHAR(56), user_id NUMBER)
```

Comment:
did you  run this locally? I think this is OK. I is also OK to change this column name to something completely different, such as `transport_mode` to completely avoid the keyword clash with Oracle. Let me know if  this is a better idea, and I can do it across all other DBs.

Reply:
> did you run this locally?

yes, without this change it wouldn't even run 😞 

> It is also OK to change this column name to something completely different, such as transport_mode to completely avoid the keyword clash with Oracle. 

That sounds perfect! 😄